packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/saml2assert/use-flow.md

```diff
@@ -38,3 +38,43 @@ Note the parameters that are being passed:
     "id_token": "eyJraWQ.....h7BYbgCzQ"
 }
 ```
+
```

Comment:
I don't think you need to add this section for SAML. The SAML assertion is not the same thing as the client ID and client secret. So maybe just leave line 24 as-is (or if you want, you can link the Base64-encoding procedure to the Client Credential procedure, but don't explicitly tell the user how to do it).

Reply:
I've just removed this stuff; no worries.